content/support-matrix/delfin/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Delfin
+menuTitle: Delfin
+description: "Support matrix data for drivers of Delfin project"
+weight: 10
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"] 
+---
+
+Delfin project aims to overcome a challenge to manage heterogeneous storage devices on a single platform.
+Delfin has standardized common API services. 
+
+ 1) Registering the devices for the following operations
```

Comment:
Please check the formatting.
Use `1.` instead of `1)`
You can refer to this [markdown guide by Github](https://guides.github.com/features/mastering-markdown/)
![image](https://user-images.githubusercontent.com/19162717/109105025-1ef07c80-7753-11eb-96f3-8a2b0c233cb9.png)

Reply:
thanks and changed as per the refernce